src/pybind/mgr/influx/module.py

```diff
@@ -233,11 +233,15 @@ def get_pg_summary_osd(self, pool_info: Dict[str, str], now: str) -> Iterator[Di
     def get_pg_summary_pool(self, pool_info: Dict[str, str], now: str) -> Iterator[Dict[str, Any]]:
         pool_sum = self.get('pg_summary')['by_pool']
         for pool_id, stats in pool_sum.items():
+            try:
+                pool_name = pool_info[pool_id]
+            except KeyError:
+                continue
```

Comment:
Let's log the exception. Something like: 
```suggestion
            try:
                pool_name = pool_info[pool_id]
            except Exception as e:
                self.log.error('Unable to find pool name for pool {}: {}'.format(pool_id, e))
                continue
```

Reply:
Thanks!